heal_swin/run_configs/segmentation/swin_synwoodscape_large_plus_AD_train_run_config.py

```diff
@@ -47,11 +47,11 @@ def get_train_run_config():
     )
     model_config = WoodscapeSegmenterSwinConfig(
         swin_transformer_config=SwinTransformerConfig(
-            window_size=[8],
+            window_size=[8, 8],
```

Comment:
Heiner noticed the same issue in his PR and changed to `window_size=8` etc. Could you do the same here and in the other config file to avoid merge conflicts later?